### PR TITLE
[measure-tools] add multi point option

### PR DESCRIPTION
**Pull request by a-gagnon**
Following discussions with itwin exp team, adding a tool setting that will allow 'stringing' measurements head to tail to make a 'line string' measurement.

Also added a debug/launch configuration to run the app.

How the ToolSettings look:
image

### CODE REVIEW

**Review by bsy-nicholasw — APPROVED** [review]

**Comment by a-gagnon** [comment]
@iTwin/viewer-components-reviewers Could one of you please have a look?

File: packages/itwin/measure-tools/src/tools/MeasureDistanceTool.ts [hunk 1]
```diff
@@ -26,14 +26,17 @@ import { MeasureTools } from "../MeasureTools";
 import { MeasureDistanceToolModel } from "../toolmodels/MeasureDistanceToolModel";
 import { SheetMeasurementsHelper } from "../api/SheetMeasurementHelper";
 import type { DrawingMetadata } from "../api/Measurement";
-import { DrawingDataCache } from "../api/DrawingTypeDataCache";
+import { type DialogItem, type DialogItemValue, type DialogPropertySyncItem, PropertyDescriptionHelper } from "@itwin/appui-abstract";
```
Comment (aruniverse) [inline packages/itwin/measure-tools/src/tools/MeasureDistanceTool.ts:29]: fyi @ben-polinsky

**Review by aruniverse — COMMENTED** [review]
can you add screenshots of what this looks like in the pr description?

**Comment by a-gagnon** [comment]
> can you add screenshots of what this looks like in the pr description?

Done

**Review by aruniverse — APPROVED** [review]

**Comment by a-gagnon** [comment]
@aruniverse I restarted the merge/CI build this morning. Can you have someone look at the issues arising with the e2e tests? It timed out in the last pipeline run (+120mins) and currently there are warnings about memory usage.

**Comment by saskliutas** [comment]
@a-gagnon It looks like e2e tests are failing because of your changes in 'test-viewer' to hide toolbar settings. Are those changes intended or left after local testing?

If they are intended I will update e2e screenshots.

**Comment by a-gagnon** [comment]
@saskliutas Oh, I see. It'd be great if you could update the screenshots.

The change is intended and, quite frankly, I'm surprised there's a way to turn off tool settings in the first place.
Apologies, I didn't expect the test-app to be used in e2e tests, I thought it was merely for development purposes.

**Pull request merged**